Makefile

```diff
@@ -59,18 +64,20 @@ test: test-style test-check test-unit test-functional
 test-check:
 	cd rootfs && python manage.py check
 
-test-style:
-	cd rootfs && flake8 --show-pep8 --show-source
+test-style: docker-build-dev
+	docker run --rm ${DEV_IMAGE} flake8 --show-pep8 --show-source
 	${SHELLCHECK_PREFIX} $(SHELL_SCRIPTS)
 
-test-unit:
-	cd rootfs \
-		&& coverage run manage.py test --settings=api.settings.testing --noinput registry api scheduler.tests \
-		&& coverage report -m
+test-unit: start-postgres do-test-unit stop-postgres
+do-test-unit: docker-build-dev
```

Comment:
I'd propose we remove the `docker-build-dev` dependency here (as, if it fails, the `deispg` stop logic won't be executed) and place it into the above `test-unit` target, to look like:
```
test-unit: docker-build-dev start-postgres do-test-unit stop-postgres
do-test-unit: 
...
```
(and same for the `-quick` variants below)